### ISSUE TITLE
Django 4.2 required despite readme specifying 3.2 support

### ISSUE THREAD

**Issue by petrklus** [mbi/django-simple-captcha#222]
See Django 4.2 at:

https://github.com/mbi/django-simple-captcha/blob/master/setup.py#L35

Expectation woudl be to see Django 3.2 there, as in version 0.5.18

**Comment by mbi** [comment]
Yes, that looks like a mistake, I'm releasing a new version in a few minutes to address this...

**Comment by mbi** [comment]
Done https://pypi.org/project/django-simple-captcha/0.5.20/
